### PR TITLE
method1 was added

### PR DESCRIPTION
**Pull request by Pablito33xD**
I have added a function

### CODE REVIEW

**Comment by Brayan721GTX** [comment]
The print method doesn't have parameters.

**Comment by Pablito33xD** [comment]
Ok, I'll add

File: main.dart [hunk 1]
```diff
@@ -2,4 +2,8 @@ class Class1 {
   Class1() {
     print("Class1");
   }
+
+  void method1() {
+    print();
```
Comment (Brayan721GTX) [inline main.dart]: It doesn't have parameter!

**Review by Brayan721GTX — CHANGES_REQUESTED** [review]
Check out this

File: main.dart (hunk 1 above)
Comment (Pablito33xD) [reply main.dart]: I see, I'll fix it

Comment (Pablito33xD) [reply main.dart]: I've fixed it

**Review by Pablito33xD — COMMENTED** [review]
I've fixed it

**Pull request merged**